packages/components/src/internal/components/samples/SampleDetailEditing.tsx

```diff
@@ -96,30 +89,15 @@ export class SampleDetailEditing extends PureComponent<Props, State> {
     };
 
     getRow = (): Record<string, any> => {
-        const { queryGridModel, queryModel } = this.props;
-
-        if (queryModel) {
-            return queryModel.getRow() ?? {};
-        } else {
-            return queryGridModel.getRow().toJS();
-        }
+        const { queryModel } = this.props;
+        return queryModel.getRow() ?? {};
     };
 
     getUpdateDisplayColumns = (isAliquot: boolean): GroupedSampleDisplayColumns => {
-        const { queryGridModel, queryModel } = this.props;
+        const { queryModel } = this.props;
         const { sampleTypeDomainFields } = this.state;
-
-        let displayColumns: List<QueryColumn>;
-        let updateColumns: List<QueryColumn>;
-
-        if (queryModel) {
-            displayColumns = List(queryModel.detailColumns);
-            updateColumns = List(queryModel.updateColumns);
-        } else {
-            displayColumns = queryGridModel.getDetailsDisplayColumns();
-            updateColumns = queryGridModel.getUpdateDisplayColumns();
-        }
-
+        const displayColumns = List(queryModel.detailColumns);
```

Comment:
Now that we don't support the `queryGridModel` variant we can stop converting to a `List` and then converting back to an `Array`. This will require updating `getGroupedSampleDisplayColumns` to return `QueryColumn[]` instead of `List` of columns.

Reply:
thanks